### PR TITLE
Move init script management to the single recipe

### PR DESCRIPTION
**Pull request by legal90**
IMO, there is no need to split an init script management to different recipes and support an additional attribute.
Following the [KISS](https://en.wikipedia.org/wiki/KISS_principle), let's just keep it in the single place, `configuration.rb`
It was already patricianly implemented in #23 

cc: @gsreynolds

### CODE REVIEW

**Comment by gsreynolds** [comment]
:+1: no problem at all, I only created a systemd recipe following the convention established by having a sysv recipe. Happy for this to be simplified.

**Comment by mvdkleijn** [comment]
Looks good and I like KISS

File: recipes/default.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 settings = Jira.settings(node)
 
 include_recipe 'chef_jira::database' if settings['database']['host'] == 'localhost'
-include_recipe "chef_jira::#{node['jira']['install_type']}"
```
Comment (legal90) [inline recipes/default.rb]: Oops, sorry, I've made a mistake here. :cry: 
@mvdkleijn Could you please roll back this line?

Comment (mvdkleijn) [inline recipes/default.rb]: Done. Missed that myself :smile: Too busy today I guess.

Comment (legal90) [inline recipes/default.rb]: Yep, me too. I've just messed `install_type` with `init_type` :)
Thanks!

**Pull request merged**